example_files/residential/clothes_dryer.tsv

```diff
@@ -1,7 +1,12 @@
-Dependency=Template Month	Dependency=Template Year	Dependency=clothes_dryer_fuel_type	clothes_dryer_efficiency	Source
-			lb/kWh	
-Sep	2020	electricity	3.93	"ResStock ""Electric, Premium, EnergyStar 100% Usage"""
-Sep	2020	natural gas	3.03	"ResStock ""Gas, Premium, 100% Usage"""
-Sep	2020	fuel oil	3.03	Engineering judgment
-Sep	2020	propane	3.03	Engineering judgment
-Sep	2020	wood	3.03	Engineering judgment
+Dependency=Template Month	Dependency=Template Year	Dependency=clothes_dryer_fuel_type	clothes_dryer_location	clothes_dryer_efficiency	Source
+				lb/kWh	
+Sep	2020	electricity	auto	3.93	"ResStock ""Electric, Premium, EnergyStar 100% Usage"""
+Sep	2020	natural gas	auto	3.03	"ResStock ""Gas, Premium, 100% Usage"""
+Sep	2020	fuel oil	auto	3.03	Engineering judgment
+Sep	2020	propane	auto	3.03	Engineering judgment
+Sep	2020	wood	auto	3.03	Engineering judgment
+Apr	2022	electricity	none	3.93	Disadvantaged community
+Apr	2022	natural gas	none	3.03	Disadvantaged community
+Apr	2022	fuel oil	none	3.03	Disadvantaged community
+Apr	2022	propane	none	3.03	Disadvantaged community
+Apr	2022	wood	none	3.03	Disadvantaged community
```

Comment:
Good use of these tsv files! Sort of demonstrates the extensibility of this approach. I suppose however "Apr 2022" doesn't exactly describe "no appliances", but maybe that's a deficiency of the design...

Reply:
Can be overloaded.

Reply:
Should I rename this as "Disadvantaged Community" or something similar?
So that the full template name would be ""Residential IECC 2018 - Customizable Template Disadvantaged Community""

Reply:
the docs.urbanopt.net , under the residential workflows section